pkg/render/fluentd.go

```diff
@@ -170,6 +173,8 @@ type FluentdConfiguration struct {
 	UsePSP bool
 	// Whether to use User provided certificate or not.
 	UseSyslogCertificate bool
+
+	EKSLogForwarderKeyPair certificatemanagement.KeyPairInterface
```

Comment:
Worth a comment explaining what this is - i.e., "This is the keypair containing the certificate presented by EKS LogForwarder when communicating with Linseed"